app/helpers/needs_helper.rb

```diff
@@ -22,6 +22,21 @@ def needs
      'Staying Social', 'Prescription pickups', 'Book drops and entertainment', 'Dog walking']
   end
 
+  def note_category_display(note_category_id)
+    note_category = needs_note_categories.find { |category| category[:id] == note_category_id }
+    note_category.present? ? note_category[:display] : 'Note'
+  end
+
+  def needs_note_categories
+    [
+      # prefix 'phone_' makes reporting easier
+      { id: 'note', display: 'Note' },
```

Comment:
Does the fact that other categories can also have a note/description make this id less clear? I wonder if an empty/null category makes more sense? Not a big deal either way.